### DISCUSSION TITLE
Restrict certain service name on Jaeger query UI and only show required service name

### DISCUSSION THREAD

**Discussion by madeinindiadot** [jaegertracing/jaeger#3432] [Q&A]
I have Jaeger sending traces to the elastic search and using the same index for viewing the traces using jaeger query UI. But the UI shows so many service names, would like to restrict or hide unnecessary service name and show only few service names there. Apart from auto complete from typing in the service name box is there any other way to restrict/hide service names in the UI

**Comment by madeinindiadot** [comment]
@yurishkuro

**Comment by pavolloffay** [comment]
There is no way to white/back list service names in the Jaeger UI.
